isos/yubikey/default.nix

```diff
@@ -17,7 +19,6 @@ with pkgs; {
     # Pre-req Packages
     gnupg
     pinentry-curses
-    pinentry-qt
```

Comment:
nice, i bet this reduced the size quite a bit

Reply:
Ya theres a few more of these in the pkg list but we'll need to go through them to see whats leading to come the UI dependencies that end up on the img.

A step in the right direction though!